prac_05/wimbledon.py

```diff
@@ -0,0 +1,53 @@
+"""
+CP1404 Practical 5
+Game, Set, Match
+Estimate: 1 second
+Actual: 0.5 second
+"""
+
+FILENAME = "wimbledon.csv"
+COUNTRY_INDEX = 1
+CHAMPION_INDEX = 2
+
+
+def main():
+    """Read, Process, and Display Wimbledon data"""
+    records = get_records(FILENAME)
+    champion_to_wins, countries = process_records(records)
+    display_results(champion_to_wins, countries)
+
+
+def get_records(filename):
+    """Read data into records list"""
+    records = []
+    with open(filename, "r", encoding="utf-8-sig") as in_file:
+        in_file.readline()
+        for line in in_file:
+            parts = line.strip().split(",")
+            records.append(parts)
+    return records
+
+
+def process_records(records):
+    """Process records into countries set and champion_to_wins dictionary"""
+    countries = set()
+    champion_to_wins = {}
```

Comment:
Would it be better to use the name "champion_to_count" for this dictionary?

Reply:
Great point